staging/cse/windows/README

```diff
@@ -12,6 +12,11 @@
 # AKS Windows CSE Scripts Package
 All files except *.test.ps1 and README will be published in AKS Windows CSE Scripts Package.
 
+## v0.0.24
```

Comment:
@AbelHu does this list look good to you?

Reply:
LGTM